app/src/main/java/com/kennedy/peter/bloquery/ui/activity/LoginActivity.java

```diff
@@ -40,11 +41,21 @@ protected void onCreate(Bundle savedInstanceState) {
         loginButton.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
-                Toast.makeText(getApplicationContext(), "LOGIN CLICKED", Toast.LENGTH_SHORT).show();
+                Firebase.AuthResultHandler handler = new Firebase.AuthResultHandler() {
+                    @Override
+                    public void onAuthenticated(AuthData authData) {
+                        Toast.makeText(LoginActivity.this, "UID: " + authData.getUid(), Toast.LENGTH_SHORT).show();
+                    }
+
+                    @Override
+                    public void onAuthenticationError(FirebaseError firebaseError) {
+                        Toast.makeText(LoginActivity.this, "Error: " + firebaseError, Toast.LENGTH_SHORT).show();
+                    }
+                };
+                firebaseManager.logIn("fakestuff420@gmail.com", "q1w2e3r4", handler);
```

Comment:
I assume this is now just for testing and it'll be replaced with the actual credentials that the user inputs.